ArtOfIllusion/src/artofillusion/Scene.java

```diff
@@ -455,27 +454,36 @@ public void addObject(Object3D obj, CoordinateSystem coords, String name, UndoRe
   public void addObject(ObjectInfo info, UndoRecord undo)
   {
     addObject(info, objects.size(), undo);
-    updateSelectionInfo();
   }
 
+  /** 
+   * Add a new object to the scene.  No undo record is added
+   */
+  public void addObject(ObjectInfo info)
```

Comment:
I'm not sure that we want another method here. Adding an object without an `UndoRecord` should be rare, and the odd thing to do.

Reply:
Fixed